static/js/search.js

```diff
@@ -0,0 +1,84 @@
+var lunrIndex, pagesIndex;
+
+// Initialize lunrjs using our generated index file
+function initLunr() {
+    // First retrieve the index file
+    $.getJSON("/json/search.json")
```

Comment:
Seems does not work if baseurl = "https://foo/bar" 
GET https://foo/json/search.json instead of https://foo/bar/json/search.json